libindy/tests/payments.rs

```diff
@@ -1724,4 +1727,36 @@ mod medium_cases {
             utils::tear_down_with_wallet(wallet_handle, "parse_verify_payment_response_works_for_generic_error", &wallet_config);
         }
     }
+
+    mod sign_with_address {
```

Comment:
Please add more tests. Just one test per function doesn't seems like TDD process...
I suggest to add some non-happy-path tests at least.

Reply:
I already have an unhappy path. I've added more one more signing test. Most of the tests in this module only have 1

Reply:
> Most of the tests in this module only have 1
It's incorrect: most of tests have only 1 happy tests and multiply tests for failure scenarios.
@mikelodder7 I think you haven't noticed the second part of tests in the file. You can find a lot of examples for unhappy path in the section `medium_cases` https://github.com/hyperledger/indy-sdk/blob/master/libindy/tests/payments.rs#L656

Reply:
I would also like to see more tests